### ISSUE TITLE
Include sticky headers in rendering html tables 

### ISSUE THREAD

**Issue by paulroth3d** [paulroth3d/jupyter-ijavascript-utils#39]
https://css-tricks.com/position-sticky-and-table-headers/

**Comment by paulroth3d** [comment]
Included in v1.27.1

**Comment by paulroth3d** [comment]
You can set the height of the table with the [`.height(css)` method on the TableGenerator class](https://jupyter-ijavascript-utils.onrender.com/TableGenerator.html#height), prior to render

```
//-- fetch the dataset
utils.datasets.fetch('cars.json').then(results => cars = results);
```

```
//-- render the table with a specific height (defaults to `50vh`)
utils.table(cars)
  .height('300px')
  .render();
```

![Screenshot](https://jupyter-ijavascript-utils.onrender.com/img/renderTableFrozenHeaders.jpg)
